bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/bridge/slip/io/SSLconnection.java

```diff
@@ -247,67 +241,39 @@ synchronized boolean available() throws IOException {
      *
      * @throws java.io.IOException in case of a communication I/O failure.
+     *             But actually eats all exceptions to ensure sure that all shutdown code is executed
      */
-    synchronized void close() throws IOException {
+    @Override
+    public synchronized void close() throws IOException {
         logger.debug("close() called.");
-        ready = false;
-        logger.info("Shutting down Velux bridge connection.");
-        // Just for avoidance of Potential null pointer access
         DataInputStreamWithTimeout dInX = dIn;
         if (dInX != null) {
-            dInX.close();
-            dIn = null;
+            try {
+                dInX.close();
+            } catch (IOException e) {
+                // eat the exception so the following will always be executed
+            }
         }
-        // Just for avoidance of Potential null pointer access
         DataOutputStream dOutX = dOut;
         if (dOutX != null) {
-            dOutX.close();
-            dOut = null;
+            try {
+                dOutX.close();
+            } catch (IOException e) {
+                // eat the exception so the following will always be executed
+            }
         }
-        // Just for avoidance of Potential null pointer access
         SSLSocket socketX = socket;
         if (socketX != null) {
-            socketX.close();
-            socket = null;
-        }
-        logger.trace("close() finished.");
-    }
-
-    /**
-     * Parameter modification.
-     *
-     * @param timeoutMSecs the maximum duration in milliseconds for read operations.
-     */
-    void setTimeout(int timeoutMSecs) {
-        logger.debug("setTimeout() set timeout to {} milliseconds.", timeoutMSecs);
-        ioTimeoutMSecs = timeoutMSecs;
-    }
-
-    /**
-     * Method to flush the input buffer.
-     *
-     * @throws java.io.IOException in case of a communication I/O failure.
-     */
-    private void flushReadBufffer() throws IOException {
-        logger.trace("flushReadBuffer() called.");
-        DataInputStreamWithTimeout dInX = dIn;
-        if (!ready || (dInX == null)) {
-            throw new IOException();
-        }
-        int byteCount = dInX.available();
-        if (byteCount > 0) {
-            byte[] byteArray = new byte[byteCount];
-            dInX.readFully(byteArray);
-            if (logger.isTraceEnabled()) {
-                StringBuilder stringBuilder = new StringBuilder();
-                for (byte currByte : byteArray) {
-                    stringBuilder.append(String.format("%02X ", currByte));
-                }
-                logger.trace("flushReadBuffer(): discarded {} unexpected bytes in the input buffer: {}", byteCount,
-                        stringBuilder.toString());
-            } else {
-                logger.warn("flushReadBuffer(): discarded {} unexpected bytes in the input buffer", byteCount);
+            logger.info("Shutting down Velux bridge connection.");
```

Comment:
Again @fwolter I must disagree with you on this specific case. The case relates to the user issuing a **remote reboot command** to the Bridge. Under normal circumstances this would never need to be done. The command is only implemented because some users experience a lock up of the Bridge. So remote reboot is a really exceptional case, and therefore it must definitely be logged at 'INFO' (if not even higher..)